event hubs migration guide.md

```diff
@@ -0,0 +1,81 @@
+# Migration Guide (EventHubs 2 to 5)
+
+## Class changes
+
+`EventHubClient` has been split into two clients - `EventHubProducerClient` (for sending
+messages) and `EventHubConsumerClient` (for subscribing to messages).
+
+| In v2                                       | Equivalent in v5                                                 | Sample |
+|---------------------------------------------|------------------------------------------------------------------|--------|
+| `EventHubClient.createFromConnectionString` | `new EventHubProducerClient()` or `new EventHubConsumerClient()` | [All](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/)
+| `EventHubClient.receive`                    | `EventHubConsumerClient.subscribe`                               | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts) |
+| `EventHubClient.receiveBatch`               | Removed in favor of `EventHubConsumerClient.subscribe`           | [receiveEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/receiveEvents.ts) |
+| `EventHubClient.createSender`               | `EventHubProducerClient.createProducer`                          | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) |
```

Comment:
```suggestion
| `EventHubClient.createSender`               | `new EventHubProducerClient`                          | [sendEvents](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/sendEvents.ts) |
```
We require instantiating the producer client. Do you know which parameters changed from v2 to v5 as well? I think we've made some required, some optional, and renamed others (epoch is now ownerLevel for instance)

Reply:
Might be good to identify which options we think of as key - quite a few things shifted around so I'd really like to stick to documenting the core scenarios if possible.

As a part of this though - if we decide that these are important parameters we should probably have a sample demonstrating their usage regardless.